openbas-api/src/main/java/io/openbas/rest/inject/service/ExecutableInjectService.java

```diff
@@ -0,0 +1,139 @@
+package io.openbas.rest.inject.service;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.openbas.database.model.*;
+import io.openbas.database.repository.InjectRepository;
+import io.openbas.rest.exception.ElementNotFoundException;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class ExecutableInjectService {
+
+  private final InjectRepository injectRepository;
+
+  private List<String> getArgumentsFromCommandLines(String command) {
+    Pattern pattern = Pattern.compile("#\\{([^#{}]+)}");
```

Comment:
Pattern compilation is cpu consuming (https://rules.sonarsource.com/java/tag/performance/RSPEC-4248/) You should pre compile it if possible.